src/main/scala/tests/typesSignatures.scala

```diff
@@ -4,8 +4,12 @@ package typesSignatures
 class A
 {
   type A = Int
-  type B[+T] = Seq[T]
-  type C[A, B <: A] = Seq[B]
+  // type B[+T] = Seq[T] It's translated into:
+  type B = [T] =>> Seq[T]
+  //variance is checked by compiler and info about it has no use for developer 
+  
+  //type C[A, B <: A] = Seq[B] Equivalent form:
+  type C = [A, B <: A] =>> Seq[B]
```

Comment:
Can we keep the old tests? If we don't, we risk regressing on them.

Reply:
There's no information about how `type C` is defined in source file (I mean if user used shorter syntax or type lambda). In TASTY we always see it as TypeLambda. I needed to decide how I want to show it. I know that this case could be shown in shorter form, but the decision how I parsed type tree applies to many other cases and I didn't want to corrupt them.

Reply:
The rule I used is that TypeLambda for abstract/opaque type is represented in shorter form, but for regular types it's shown as TypeLambda

Reply:
Hm, that's correct - I thought Dotty REPL could distinguish between the two, but it cannot do that as well. In that case I'd actually do what the REPL does and always display type members in the `type F[+A] = List[A]` form. We can test type lambda rendering by creating aliases like

```scala
type Id[T] = T
type G = Id[[A] =>> List[A]]
```

Reply:
Ok, done.